lms/djangoapps/grades/constants.py

```diff
@@ -13,6 +13,6 @@ class ScoreDatabaseTableEnum(object):
 
 
 class GradeOverrideFeatureEnum(object):
-    proctoring = 'PROCTORING'
-    gradebook = 'GRADEBOOK'
+    proctoring = u'PROCTORING'
+    gradebook = u'GRADEBOOK'
     grade_import = 'grade-import'
```

Comment:
Any reason for leaving just this one non-prefixed?

Reply:
It did not lead to any type mismatches, so I didn't change this.

Reply:
Looks like we don't use it anywhere?  Or maybe it's only used from a plugin (e.g. edx-bulk grades)?

Reply:
ya, it doesn't seem to be used anywhere, any opposition to just removing it?

Reply:
@matthugs We added this enum value fairly recently - do you remember if there's a specific use we had in mind for it?  I can't find any actual usage of it in edx-platform or edx-bulk-grades (or anywhere else in our github org).

Reply:
I'm tabling this conversation for now and merging the pull request.